### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by HarryPhoon**
Addressing #189 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,33 @@
-Citadel
-=======
+# Argentum Age
 
-A turn based strategy game based on the Anura engine
+[<img src="http://argentumage.com/favicon-32.png" width="20"> Official website](https://argentumage.com) | [<img src="https://cdnjs.cloudflare.com/ajax/libs/webicons/2.0.0/webicons/webicon-twitter.svg" width="20"> Twitter](https://twitter.com/argentum_age) | [<img src="https://cdnjs.cloudflare.com/ajax/libs/webicons/2.0.0/webicons/webicon-reddit.svg" width="20"> Reddit](https://www.reddit.com/r/ArgentumAge/) | [<img src="https://cdnjs.cloudflare.com/ajax/libs/webicons/2.0.0/webicons/webicon-facebook.svg" width="20"> Facebook](https://www.facebook.com/ArgentLark) | [<img src="https://discordapp.com/assets/f8389ca1a741a115313bede9ac02e2c0.svg" width="22"> Discord](https://discord.gg/0s6mSoZ1vJPlvcIw)
```
Comment (galegosimpatico) [inline README.md]: Here a line break would do the same as a single space. So you can break a huge line into five smaller lines to no render disadvantage but easier future diff maintenance.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+To get involved as a developer, check out this wiki page: [Getting started as a developer](https://github.com/davewx7/citadel/wiki/GettingStartedAsADeveloper).
+
+If you want to contribute some art, check out this page: [Argentum Age card art guidelines](https://github.com/davewx7/citadel/wiki/Argentum-Age-Card-Art-Guidelines).
+
+The game development is currently a bit stalled because we'll be transitioning from the Anura engine to Unity, but if you still want to contribute to current codebase we will be more than happy.
+
+## Useful links
+
+- [Anura engine GitHub](https://github.com/anura-engine/anura)
+- [Anura Engine Documentation](https://github.com/frogatto/frogatto/wiki)
```
Comment (galegosimpatico) [inline README.md]: This is the Frogatto wiki, not the Anura wiki. The Anura wiki is split in the Frogatto wiki and the Anura wiki itself. One link is missing.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+- [Frogatto & Friends GitHub](https://github.com/frogatto/frogatto)
+- [List of all cards](http://theargentlark.com/david/cards/all-cards.html)
```
Comment (galegosimpatico) [inline README.md]: This is not gold, and those are not _all_ cards anyway.

Comment (HarryPhoon) [reply README.md]: Ok, this doesn't need to be there. Removing these two lines.

File: README.md (hunk 2 above)
Comment (HarryPhoon) [reply README.md]: Ok, I forgot about the small of the documentation that is in the Anura wiki.

I'll add the link.

File: README.md (hunk 1 above)
Comment (HarryPhoon) [reply README.md]: Ok.

File: README.md (hunk 3 above)
Comment (galegosimpatico) [reply README.md]: Frogatto **is** gold, much more than Argentum Age.
:)

**Pull request merged**